.travis.yml

```diff
@@ -16,21 +16,17 @@ matrix:
     - php: 5.3.3
       env: USE_DEPS="lowest"
     - php: 5.6
-      env: SYMFONY_VERSION="2.3.*"
-    - php: 5.6
-      env: SYMFONY_VERSION="2.4.*" SYMFONY_DEPRECATIONS_HELPER="weak"
-    - php: 5.6
-      env: SYMFONY_VERSION="2.5.*" SYMFONY_DEPRECATIONS_HELPER="weak"
-    - php: 5.6
       env: SYMFONY_VERSION="2.6.*" SYMFONY_DEPRECATIONS_HELPER="weak"
     - php: 5.6
       env: SYMFONY_VERSION="2.8.*" SYMFONY_DEPRECATIONS_HELPER="weak"
```

Comment:
2.8 should not allow weak deprecations (otherwise, the code is not Symfony 3 ready)